### PR TITLE
PR for Adding tests for installing and deleting virtual cluster from kubectl

### PR DESCRIPTION
**Pull request by sowmyav27**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind test

- But this is for testing the GHA with some minor changes.
- Original PR: https://github.com/loft-sh/vcluster/pull/2011
- If this works, depending on how we want to proceed, I might close of these PRs
- fixed comments from here - https://github.com/loft-sh/vcluster/pull/2011#pullrequestreview-2213293663

**Details:**
- Implemented a module for running the install and delete commands for virtual cluster through helm and kubectl in the `e2e.yaml` GHA workflow file. 
- Added bash scripts for the actions - helm and kubectl

**To-do in next PR** 
- once this PR is merged - install and delete using argoCD and terraform

**What does this pull request do?**
Adds a test - install and delete vcluster using kubectl and helm

**Please provide a short message that should be published in the vcluster release notes**
N/A

**What else do we need to know?**

### CODE REVIEW

**Review by FabianKramm — CHANGES_REQUESTED** [review]
Not sure if this is doing what we want, within our e2e test pipeline we want. The test itself looks good, but the problem is that the e2e pipeline itself is creating a vcluster as well via the regular method, it seems to me you only want to install & delete via kubectl and not want to have that other vcluster created as well. You either need to adjust the e2e pipeline itself for this (.github/workflows/e2e.yaml) or need to create a separate pipeline.

File: test/e2e_vcluster_install/deploy_and_delete_vcluster.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		err = kubectlCmd.Run()
+		framework.ExpectNoError(err)
+
+		gomega.Eventually(func() bool {
+
+			checkCmd := exec.Command("vcluster", "list")
+			output, err := checkCmd.CombinedOutput()
+			framework.ExpectNoError(err)
+			return err == nil && strings.Contains(string(output), vClusterName) && strings.Contains(string(output), "Running")
+		}, pollingDurationLong, pollingInterval).Should(gomega.BeTrue(), "Virtual cluster failed to come up Active")
```
Comment (zerbitx) [inline test/e2e_vcluster_install/deploy_and_delete_vcluster.go:50]: nit: Recommend using the "With" methods like [these](https://github.com/loft-sh/vcluster/blob/main/test/e2e_plugin/plugin/plugin.go#L30-L31)

**Review by FabianKramm — APPROVED** [review]

**Review by FabianKramm — CHANGES_REQUESTED** [review]
I tihnk whats weird is that we have all these different dimensions still for your new test. For what distros and dimensions do you want to run your test?

**Review by FabianKramm — APPROVED** [review]
@sowmyav27 LGTM! Thanks for the PR!

**Pull request merged**